sources/packages/api/src-sql/Applications/Add-col-program-year-id.sql

```diff
@@ -0,0 +1,13 @@
+-- Create program_year_id
+ALTER TABLE
+    sims.applications
+ADD
+    COLUMN IF NOT EXISTS program_year_id INT NOT NULL REFERENCES sims.program_years(id) DEFAULT 2;
+
+COMMENT ON COLUMN sims.applications.program_year_id IS 'References the program year related to the application, this will be saved only when the ProgramYear is active. The default 2 is the Program Year 2022-2023, this is used to ensure not null constraint';
```

Comment:
The comment about the Id 2 used should not be present in the column comments because it will no longer exists after the "DROP DEFAULT" right below.
Please, add a comment to the migration script, not to the column comment.